node/config/def.go

```diff
@@ -66,6 +66,21 @@ type DealmakingConfig struct {
 
 	Filter          string
 	RetrievalFilter string
+
+	// If `UseDynamicRetrievalPricing` is set to true(default is false), we will use the `DynamicRetrievalPricingFunc` script
+	// to dynamically price retrieval deals.
+	UseDynamicRetrievalPricing  bool
```

Comment:
Suggestion:


```go
RetrievalPricing *RetrievalPricing

type RetrievalPricing struct {
    Strategy string // possible values: "default", "external"

    External *RetrievalPricingExternal
    Default  *RetrievalPricingDefault
}

type RetrievalPricingExternal struct {
    Path string
}

type RetrievalPricingDefault struct {
    VerifiedDealsFreeTransfer bool
}
```

This would lead to this configuration:

```toml
[DealmakingConfig]
RetrievalPricing.Strategy = "default"
RetrievalPricing.Default.VerifiedDealsFreeTransfer = true
```

or

```toml
[DealmakingConfig]
RetrievalPricing.Strategy = "external"
RetrievalPricing.External.Path = "/path/to/script"
```